packages/mux-player/src/types.d.ts

```diff
@@ -33,6 +34,7 @@ export type MuxTemplateProps = Partial<MuxPlayerProps> & {
   metadataVideoTitle: string;
   metadataViewerUserId: string;
   noHotKeys: boolean;
+  hotKeys: AttributeTokenList;
```

Comment:
issue(blocker): This is inaccurate.

Reply:
oops, forgot to update this one. Good callout.

Reply:
cool that tracks. Resolving.